src/main/java/cz/kominekjan/disenchantment/commands/impl/Enchantments.java

```diff
@@ -9,70 +12,110 @@
+                    return Config.getEnchantmentsStatus();
+                }
+
+                @Override
+                public void setEnchantmentsStatus(Enchantment enchantment, EnchantmentStatus status) {
+                    Config.setEnchantmentsStatus(enchantment, status);
+                }
+            }
+    );
+    public static final Command bookSplittingCommand = new Command(
+            "split-enchantments",
+            new String[]{"disenchantment.all", "disenchantment.command.enchantments"},
+            "You don't have permission to use this command.",
+            new String[]{},
+            false,
+            new Enchantments() {
+                @Override
+                public Map<Enchantment, EnchantmentStatus> getEnchantmentsStatus() {
+                    return Config.getBookSplittingEnchantmentsStatus();
+                }
+
+                @Override
+                public void setEnchantmentsStatus(Enchantment enchantment, EnchantmentStatus status) {
+                    Config.setBookSplittingEnchantmentsStatus(enchantment, status);
+                }
+            }
     );
 
-    public static void execute(CommandSender s, String[] args) {
-        Map<Enchantment, Boolean> disabledEnchantments = getDisabledEnchantments();
+    public void execute(CommandSender s, String[] args) {
+        Map<Enchantment, EnchantmentStatus> enchantmentStatus = getEnchantmentsStatus();
 
         if (args.length == 1) {
             s.sendMessage(textWithPrefix("Disabled enchantments"));
             s.sendMessage("");
 
-            if (disabledEnchantments.isEmpty()) {
-                s.sendMessage(ChatColor.GRAY + "No enchantments are disabled");
+            long nonEnabledCount = enchantmentStatus.values().stream().filter(status -> !EnchantmentStatus.ENABLED.equals(status)).count();
+
+            if (nonEnabledCount == 0) {
+                s.sendMessage(ChatColor.GRAY + "No enchantments are disabled or set to being kept");
                 return;
             }
 
-            for (Map.Entry<Enchantment, Boolean> enchantment : disabledEnchantments.entrySet()) {
-                String builder = "";
-                builder += ChatColor.RED + "[" + (enchantment.getValue() ? " keep " : "cancel") + "] ";
-                builder += ChatColor.GRAY + enchantment.getKey().getKey().getKey();
-                s.sendMessage(builder);
+            for (Map.Entry<Enchantment, EnchantmentStatus> enchantment : enchantmentStatus.entrySet()) {
+                EnchantmentStatus status = enchantment.getValue();
+                if(EnchantmentStatus.ENABLED.equals(status)) continue;
+
+                StringBuilder builder = new StringBuilder();
+
+                switch (status){
+                    case DISABLED -> builder.append(ChatColor.RED).append("[cancel] ");
+                    case KEEP -> builder.append(ChatColor.GOLD).append("[ keep ] ");
```

Comment:
the spaces between `[ ]` should be there or not?

Reply:
As per deleted line 
`builder += ChatColor.RED + "[" + (enchantment.getValue() ? " keep " : "cancel") + "] ";`. 
yes only for keep.